docs/guides/nrfconnect_examples_software_update.md

```diff
@@ -0,0 +1,129 @@
+# Performing Device Firmware Upgrade in the nRF Connect SDK examples
+
+The following examples for the development kits from Nordic Semiconductor
+support over-the-air Device Firmware Upgrade:
+
+-   [CHIP nRF Connect Lock Example Application](../../examples/lock-app/nrfconnect/README.md)
+-   [CHIP nRF Connect Lighting Example Application](../../examples/lighting-app/nrfconnect/README.md)
+
+The
```

Comment:
Do we need to explicitly mention which examples do not support it, after you listed which support? I would remove it, as you didn't list all nRF Connect examples below anyway.